### PR TITLE
ci: add e2e testing

### PR DESCRIPTION
**Pull request by erikgb**
This enables the e2e tests based on kuttl in the CI workflow, using a k3d/k3s Kubernetes cluster.

The private image test will have to be disabled for now, as we have no private image to test with - with known pull credentials that can be included in the project. I will create an issue to follow this up. As kuttl does not currently provide any feature to selectively disable tests, ref. https://github.com/kudobuilder/kuttl/issues/400, I have moved that test to a sibling directory.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -100,3 +100,30 @@ jobs:
...
+      - run: |
+          k3d image import --cluster image-scanner /tmp/controller-image.tar
+          make deploy-dependencies deploy
+      - uses: jaxxstorm/action-install-gh-release@v1
+        with:
+          repo: kudobuilder/kuttl
+          tag: v0.15.0
+          cache: enable
+      - run: |
+          kubectl kuttl test --config test/e2e-config/kuttl-test.yaml
```
Comment (bendikp) [inline .github/workflows/ci.yml:129]: Can we use the make target here instead?
```suggestion
          make e2e-test
```

**Comment by mikaello** [comment]
Just curious, what does `rio` in `rio_disabled` mean?

**Comment by mikaelol** [comment]
Just curious, what does `rio` in `rio_disabled` mean?

**Comment by bendikp** [comment]
> Just curious, what does rio in rio_disabled mean?

I think you missed the `scena` in `rio` 😄

**Review by mikaelol — APPROVED** [review]

**Pull request merged**